adapters/audienceNetwork/facebook.go

```diff
@@ -204,15 +197,14 @@ func (this *FacebookAdapter) modifyImp(out *openrtb.Imp) error {
 			}
 		}
 
-		/* This will get overwritten post-serialization */
-		out.Banner.W = openrtb.Uint64Ptr(0)
+		out.Banner.W = openrtb2.Int64Ptr(-1)
```

Comment:
This doesn't have to be a hack now. It's ok to use a negative number here. We're past bid request validation.